content/blog/2021-01-21-unpack-cfr.Rmd

```diff
@@ -0,0 +1,249 @@
+| 70-79            | 0.31 (0.26, 0.35) | 0.19 (0.14, 0.23) | -0.38 (-0.57, -0.17)   | 0.45 (0.43, 0.47)    | 0.18 (0.16, 0.2)       | -0.6 (-0.65, -0.54)  |
+| 80+              | 0.44 (0.39, 0.49) | 0.37 (0.32, 0.41) | -0.17 (-0.32, -0.0038) | 0.62 (0.59, 0.64)    | 0.28 (0.25, 0.31)      | -0.55 (-0.59, -0.49) |
+</center>
+**Table 2.** _Estimates of HFR and drop in HFR between April 1st and December 1st. Median and $95\%$ confidence intervals are computed using block bootstrapping. Results with inadequate support are omitted._
+
+Consistent with findings that increased age is associated with higher mortality rates (@Mahasem1327_agedeathrate), we observe that as the age of the group increases, the corresponding HFR increases. 
+Measuring treatment improvements by HFR drop (computed as $\frac{\text{HFR}_{new} - \text{HFR}_{old}}{\text{HFR}_{old}}$), we also note that larger treatment improvements between April and December are correlated with younger age. 
+Note, however, that the younger groups have small HFRs to begin with, so the opposite trend may appear when considering _absolute_ rather than _relative_ improvements. 
+Additionally, the confidence intervals for HFR drops are wider for younger age groups.
+
+Between the first two peaks (April 15th to July 15th, Table 1), the national age-stratified HFR estimates from block bootstrapping decrease by as little as $27\%$ in the 80+ age group, and as much as $37\%$ in the 30-39 age group. On the other hand, in Florida the age-stratified HFR actually _increases_ in each age group by as little as $2.9\%$ in the 80+ age group, and as much as $13\%$ in the 60-69 age group. Note that the HFR changes between peak dates in Florida are an example of Simpson's paradox, where in each age group the HFR increase, but the aggregate HFR actually decreases by $2.3\%$.
+
+Compared to peak-to-peak changes, changes across the entire time range (April 1st to December 1st, Table 2) show a more dramatic decrease. In Florida, the HFR drops by as little as $17\%$ in the 80+ age range, and as much as $42\%$ in the 60-69 age range. Nationally, the HFR drops by as little as $55\%$ in the 80+ age groups, and as much as $73\%$ in the 20-29 age group. In both Florida and national data, as the age group gets older (for those older than 40), we observe an increase in age-stratified HFR and smaller treatment improvements as measured by HFR drop.
+
+While we focus on the two peaks and the endpoints of the study time range, we also include plots of HFR estimates with uncertainty for all dates between April 1st and December 1st: 
+
+![hfr_florida](img/img1/florida_fdoh_full_est.svg)
+<p style="text-align: center;"><i>(a) Florida FDOH data</i></p>
+
+![hfr_florida](img/img_country/national_cleaned_full_est.svg)
+<p style="text-align: center;"><i>(b) United States CDC data</i></p>
```

Comment:
these captions can again go into the square brackets, otherwise `hfr_florida` will appear underneath the images

Reply:
Done.